hello-kashif.py

```diff
@@ -0,0 +1,2 @@
+name='Kashif'
```

Comment:
Even though your Python script works, my code review found this section where you set the variable name to your name, could you please help me understand more, before I approve your PR.

(Thanks for playing along Kashif ;-))